client/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_BASE_URL=http://localhost:8080
```

Comment:
You can't use this approach with our deployment system. You rely instead on the "proxy" field in package.json.